.github/workflows/docker.yaml

```
@@ -1,81 +1,67 @@
         uses: actions/checkout@v4
-
-      - id: string
-        uses: ASzc/change-string-case-action@v6
+      
+      - name: Set lowercase image name
+        id: image
+        run: echo "name=$(echo '${{ github.repository }}' | tr '[:upper:]' '[:lower:]')" >> $GITHUB_OUTPUT
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)